examples/src/main/java/com/mapbox/navigation/examples/core/IndependentRouteGenerationActivity.kt

```diff
@@ -291,17 +289,10 @@ class IndependentRouteGenerationActivity : AppCompatActivity() {
         mapboxNavigation.unregisterRouteProgressObserver(replayProgressObserver)
         mapboxNavigation.unregisterRoutesObserver(routesObserver)
         mapboxNavigation.unregisterRouteProgressObserver(routeProgressObserver)
-        binding.mapView.onStop()
     }
 
     override fun onDestroy() {
         super.onDestroy()
-        binding.mapView.onDestroy()
         mapboxNavigation.onDestroy()
     }
-
-    override fun onLowMemory() {
```

Comment:
This looks unrelated. What's the explanation here?

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/4880#discussion_r714232342

Reply:
> This looks unrelated. 

It actually is, sorry. I extracted those changes into a separate commit for a better history in git.